lib/vdsm/storage/lvmconf.py

```diff
@@ -184,3 +189,38 @@ def save(self):
     def close(self):
         log.debug("Closing LVM configuration %s", self.path)
         self.aug.close()
+
+
+def configured_value(section, option):
+    """
+    Return value configured for the option, taken into account all config
+    files or default configurations.
+    """
+    if not section:
+        raise ValueError("Section must not be empty.")
+    if not option:
+        raise ValueError("Option must not be empty.")
+
+    cmd = [constants.EXT_LVM, "lvmconfig",
+           "--typeconfig", "full",
+           f"{section}/{option}"]
+    out = commands.run(cmd).decode("utf-8").strip()
+
+    # Validate, that the output is in expected format key=value and doesn't
+    # span multiple lines.
+    if "=" not in out or "\n" in out:
+        raise se.UnexpectedLvmConfigOutput(
+            f"Unexpected output: option={section}/{option}, output={out}")
+
+    key, value = out.split("=")
```

Comment:
We want to split exactly once:

    split("=", 1)

Otherwise if the value includes "=" this call will fail:

    >>> key, value = "key=val=ue".split("=") 
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    ValueError: too many values to unpack (expected 2)

Reply:
done